client/card-readers/settings/sections/address-details.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { TextControl, SelectControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAccountBusinessSupportAddress } from '../../../data';
+
+const ReadersSettingsFields = () => {
+	const [
+		accountBusinessSupportAddress,
+		accountBusinessSupportAddressCountry,
+		accountBusinessSupportAddressLine1,
+		accountBusinessSupportAddressLine2,
+		accountBusinessSupportAddressCity,
+		accountBusinessSupportAddressPostalCode,
+		setAccountBusinessSupportAddress,
+	] = useAccountBusinessSupportAddress();
+
+	const handleAddresPropertyChange = ( property, value ) => {
```

Comment:
There seems to be a small typo here in the word "address."

Reply:
Sorry, fixed! :)